test/fixtures/types/code-sample-definitions.ts

```diff
@@ -13,6 +13,7 @@ export default [
         foo: {
           foo_id: '8d7e0b3a-b889-49a7-9164-4b71a0506a33',
           name: 'Best foo',
+          metadata: null,
```

Comment:
This was confusing to me. If we update the test fixture with a new property we should update the entire fixture (all files), e.g., all files in https://github.com/seamapi/blueprint/tree/main/test/fixtures/types

This is important because we may add future logic to validate code samples against the API spec, etc. and we need the fixture to be self consistent.

Instead of `metadata` let's name this new property something more intuitive, maybe `nullable_property`?

Reply:
Got it, will make the changes. What's the difference between https://github.com/seamapi/blueprint/blob/main/test/fixtures/types/route-schemas.ts and https://github.com/seamapi/blueprint/blob/main/test/fixtures/types/route-specs.ts? They seem to be identical.

Reply:
2f6e3abcb233e74c52ba918eb0822d7d8cb1c5b5
78eb64efdb3878a3b3fb5d6d39f34fa763e6b914
941c4eae932c279524b4b64e08364d0e3280fb82
25c392e3d9b05bb8e95be6d8a34eb01867d23f57
a290894a8e1c6a2c3391cdbb8c9e29a3c30ad9c4
1afcb4c138c2e46e29beb1e036361e48e10ccfdc

Reply:
> Got it, will make the changes. What's the difference between [main/test/fixtures/types/route-schemas.ts](https://github.com/seamapi/blueprint/blob/main/test/fixtures/types/route-schemas.ts?rgh-link-date=2024-08-07T09%3A59%3A41Z) and [main/test/fixtures/types/route-specs.ts](https://github.com/seamapi/blueprint/blob/main/test/fixtures/types/route-specs.ts?rgh-link-date=2024-08-07T09%3A59%3A41Z)? They seem to be identical.

@andrii-balitskyi This might be a mistake. I think I copied instead of renamed the file? Anyway it should be `route-specs.ts`

Reply:
7e676977b04688914bd4098aa076c14c29d1457e